2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (1, 'Eric', 'Cartman', 'ecartman@spark.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (2, 'Kenny', 'McCormick', 'kmccormick@spark.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (3, 'Kyle', 'Broflovski', 'kbroflovski@spark.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (4, 'Stan', 'Marsh', 'smarsh@spark.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (5, 'Rick', 'Sanchez', 'rsanchez@randm.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (6, 'Morty', 'Smith', 'msmith@randm.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (7, 'Summer', 'Smith', 'ssmith@randm.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (8, 'Philip', 'Fry', 'pfry@futura.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (9, 'Bender', 'Rodr√≠guez', 'brodriguez@futura.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (10, 'John', 'Zoidberg', 'jzoidberg@futura.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEE" not found; SQL statement:
INSERT INTO EMPLOYEE(ID, FIRST_NAME, LAST_NAME, EMAIL) VALUES (11, 'Hubert', 'Farnsworth', 'hfarnsworth@futura.com') [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EVENTS" not found; SQL statement:
INSERT INTO EVENTS(ID, NAME, BEGIN, END, OWNER_ID) VALUES (1, "Planet Express - Executive board meeting", "2018-12-01T09:00:00.000Z", "2018-12-01T11:00:00.000Z", 11) [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEES_SKILLS" not found; SQL statement:
INSERT INTO EMPLOYEES_SKILLS(SKILLS_ID, EMPLOYEES_ID) VALUES (1, 1) [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEES_SKILLS" not found; SQL statement:
INSERT INTO EMPLOYEES_SKILLS(SKILLS_ID, EMPLOYEES_ID) VALUES (2, 1) [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEES_SKILLS" not found; SQL statement:
INSERT INTO EMPLOYEES_SKILLS(SKILLS_ID, EMPLOYEES_ID) VALUES (3, 1) [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEES_SKILLS" not found; SQL statement:
INSERT INTO EMPLOYEES_SKILLS(SKILLS_ID, EMPLOYEES_ID) VALUES (2, 2) [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEES_SKILLS" not found; SQL statement:
INSERT INTO EMPLOYEES_SKILLS(SKILLS_ID, EMPLOYEES_ID) VALUES (3, 2) [42102-197]
2018-09-20 16:30:42 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "EMPLOYEES_SKILLS" not found; SQL statement:
INSERT INTO EMPLOYEES_SKILLS(SKILLS_ID, EMPLOYEES_ID) VALUES (3, 3) [42102-197]
2018-09-20 16:31:20 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.createStatement(JdbcConnection.java:217)
	at org.apache.tomcat.jdbc.pool.PooledConnection.validate(PooledConnection.java:536)
	at org.apache.tomcat.jdbc.pool.PooledConnection.validate(PooledConnection.java:454)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.borrowConnection(ConnectionPool.java:803)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.borrowConnection(ConnectionPool.java:652)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.getConnection(ConnectionPool.java:198)
	at org.apache.tomcat.jdbc.pool.DataSourceProxy.getConnection(DataSourceProxy.java:132)
	at org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.getConnection(DatasourceConnectionProviderImpl.java:122)
	at org.hibernate.tool.hbm2ddl.SuppliedConnectionProviderConnectionHelper.prepare(SuppliedConnectionProviderConnectionHelper.java:33)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.<init>(DatabaseExporter.java:35)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:425)
	at org.hibernate.tool.hbm2ddl.SchemaExport.drop(SchemaExport.java:375)
	at org.hibernate.tool.hbm2ddl.SchemaExport.drop(SchemaExport.java:371)
	at org.hibernate.internal.SessionFactoryImpl.close(SessionFactoryImpl.java:1081)
	at org.hibernate.jpa.internal.EntityManagerFactoryImpl.close(EntityManagerFactoryImpl.java:347)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:495)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:671)
	at com.sun.proxy.$Proxy85.close(Unknown Source)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.destroy(AbstractEntityManagerFactoryBean.java:592)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:272)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:583)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:555)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:961)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:516)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:968)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1032)
	at org.springframework.context.support.AbstractApplicationContext.doClose(AbstractApplicationContext.java:1008)
	at org.springframework.context.support.AbstractApplicationContext$2.run(AbstractApplicationContext.java:929)
2018-09-20 16:31:20 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:476)
	at org.apache.tomcat.jdbc.pool.PooledConnection.validate(PooledConnection.java:557)
	at org.apache.tomcat.jdbc.pool.PooledConnection.validate(PooledConnection.java:454)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.borrowConnection(ConnectionPool.java:803)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.borrowConnection(ConnectionPool.java:652)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.getConnection(ConnectionPool.java:198)
	at org.apache.tomcat.jdbc.pool.DataSourceProxy.getConnection(DataSourceProxy.java:132)
	at org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.getConnection(DatasourceConnectionProviderImpl.java:122)
	at org.hibernate.tool.hbm2ddl.SuppliedConnectionProviderConnectionHelper.prepare(SuppliedConnectionProviderConnectionHelper.java:33)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.<init>(DatabaseExporter.java:35)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:425)
	at org.hibernate.tool.hbm2ddl.SchemaExport.drop(SchemaExport.java:375)
	at org.hibernate.tool.hbm2ddl.SchemaExport.drop(SchemaExport.java:371)
	at org.hibernate.internal.SessionFactoryImpl.close(SessionFactoryImpl.java:1081)
	at org.hibernate.jpa.internal.EntityManagerFactoryImpl.close(EntityManagerFactoryImpl.java:347)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:495)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:671)
	at com.sun.proxy.$Proxy85.close(Unknown Source)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.destroy(AbstractEntityManagerFactoryBean.java:592)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:272)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:583)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:555)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:961)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:516)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:968)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1032)
	at org.springframework.context.support.AbstractApplicationContext.doClose(AbstractApplicationContext.java:1008)
	at org.springframework.context.support.AbstractApplicationContext$2.run(AbstractApplicationContext.java:929)
2018-09-20 16:31:20 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:476)
	at org.apache.tomcat.jdbc.pool.PooledConnection.validate(PooledConnection.java:565)
	at org.apache.tomcat.jdbc.pool.PooledConnection.validate(PooledConnection.java:454)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.borrowConnection(ConnectionPool.java:803)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.borrowConnection(ConnectionPool.java:652)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.getConnection(ConnectionPool.java:198)
	at org.apache.tomcat.jdbc.pool.DataSourceProxy.getConnection(DataSourceProxy.java:132)
	at org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.getConnection(DatasourceConnectionProviderImpl.java:122)
	at org.hibernate.tool.hbm2ddl.SuppliedConnectionProviderConnectionHelper.prepare(SuppliedConnectionProviderConnectionHelper.java:33)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.<init>(DatabaseExporter.java:35)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:425)
	at org.hibernate.tool.hbm2ddl.SchemaExport.drop(SchemaExport.java:375)
	at org.hibernate.tool.hbm2ddl.SchemaExport.drop(SchemaExport.java:371)
	at org.hibernate.internal.SessionFactoryImpl.close(SessionFactoryImpl.java:1081)
	at org.hibernate.jpa.internal.EntityManagerFactoryImpl.close(EntityManagerFactoryImpl.java:347)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:495)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:671)
	at com.sun.proxy.$Proxy85.close(Unknown Source)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.destroy(AbstractEntityManagerFactoryBean.java:592)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:272)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:583)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:555)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:961)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:516)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:968)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1032)
	at org.springframework.context.support.AbstractApplicationContext.doClose(AbstractApplicationContext.java:1008)
	at org.springframework.context.support.AbstractApplicationContext$2.run(AbstractApplicationContext.java:929)
